app/src/main/java/org/schabi/newpipe/util/Localization.java

```diff
@@ -185,17 +198,38 @@ public static String localizeWatchingCount(final Context context, final long wat
 
     public static String shortCount(final Context context, final long count) {
```

Comment:
```suggestion
    public static String shortCount(final Context context, final long count) {
        if (Build.VERSION.SDK_INT >= 24) {
            return CompactDecimalFormat.getInstance(getAppLocale(context),
                    CompactDecimalFormat.CompactStyle.SHORT).format(count);
        }
```
You can easily localize if you focus on Android 7.0 or later, which has a built-in ICU library.
Even if it is below Android 7.0, it will work if you bundle the library, but I want to avoid it because it is about 12MB.

Also in this case, I think the `short_billion`, `short_million` and `short_thousand` on Weblate need a note that they are only used for below Android 7.0.